### ISSUE TITLE
Scrolling terminal keyboard shortcuts not working on MacOS

### ISSUE THREAD

**Issue by kieferrm** [Microsoft/vscode#12809]
Testing #12685.

- VSCode Version: 1.6.0 insiders
- OS Version: MacOs

Steps to Reproduce:

1. Open a terminal
2. execute command with multipage output
3. use Command+ArrowUp to scroll a single line up
-> `9A` is printed to the console but no scrolling

Executing the command from the command palette works as expected.

Same for Scrolling a line down, or scrolling page wise.

**Comment by Tyriar** [comment]
@kieferrm do you have you have any terminal-related settings on your settings.json?

**Comment by kieferrm** [comment]
@Tyriar: No, I don't.

**Comment by Tyriar** [comment]
Cannot reproduce on my Mac, cmd+up/down scroll fine and shift+page up/down work (shift+fn+up/down).

One thing that should change is to use just page up/down for scrolling a page to be consistent with the default Mac terminal.
